WordPress/Classes/Services/PostTagService.h

```diff
@@ -45,6 +45,12 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)deleteTag:(PostTag*)tag
           forBlog:(Blog *)blog;
 
+/**
+ Commits a tag assigned to a blog
+ */
+- (void)commitTag:(PostTag*)tag
```

Comment:
`commit` as a method name works but its not one we've used anywhere else in the app. It would probably be more consistent to use `save`.